test/hotspot/jtreg/serviceability/sa/ClhsdbDumpheap.java

```diff
@@ -37,9 +37,13 @@
 
 /**
  * @test
- * @bug 8240989
+ * @bug 8240989 8336753
```

Comment:
I don't think that change makes sense because the test is not a regression test for JDK-8336753.

Reply:
I don't think that's necessary, one can always find the changeset that added a specific line/comment via the git history.

Reply:
Fair enough, now removed.